test/parallel/test-argv0-property.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+assert.ok(process.hasOwnProperty('argv0'), 'argv0 property exists');
+assert.throws(() => {
+  process.argv0 = 'anything';
+}, /read only/);
+assert.doesNotThrow(() => {
+  delete process.argv0;
+  Object.defineProperty(process, 'argv0', {value: 'some-test-value'});
+}, Error, 'argv0 is configurable');
+assert.strictEqual(process.argv0, 'some-test-value', 'new property assigned');
```

Comment:
Just a tiny nit, but I’ve recently made a PR that did removed a ton of `assert` message parameters in our tests; if they are just literals, they never really provide more information than the stack trace itself in the case of a failure, but the expected/actual values do.

(I also have a lint rule for disabling these, but it’s not enabled because there’s still a lot of that in our codebase… sorry!)